source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -126,7 +126,7 @@
         var self = this;
         this.userView = data.UserView;
         this.userLogged = data.UserLogged;
-        this.hasAverageColumn = !data.Evaluation.Finished && (this.userView == 1 || this.userView == 3);
```

Comment:
@AlphaGit  good concern, I'm trying to figure out the idea of the average. I'm asking because I believe that the idea is to show the company score.

I'm not will merge this branch until know what to show